appengine/src/main/java/com/google/googleinterns/gscribe/modules/services/ExamGenerationServiceModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.googleinterns.gscribe.modules.services;
+
+import com.google.googleinterns.gscribe.services.ExamGenerationService;
+import com.google.googleinterns.gscribe.services.impl.ExamGenerationServiceImpl;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+
+public class ExamGenerationServiceModule extends AbstractModule {
+
+    @Provides
+    @Singleton
+    ExamGenerationService examGenerationServiceProvider() {
```

Comment:
All of these services can be part of a single module.

Reply:
So basically all 7 module classes joined into 1 service module class, or 1 module class for exam service and 1 for token module class?